rules/TypeDeclaration/Rector/Param/ParamTypeFromStrictTypedPropertyRector.php

```diff
@@ -168,28 +169,23 @@ private function matchPropertySingleTypeNode(PropertyFetch $propertyFetch): ?Nod
         return $this->staticTypeMapper->mapPHPStanTypeToPhpParserNode($propertyType, TypeKind::PROPERTY);
     }
 
-    private function hasTypeChangedBeforeAssign(Assign $assign, string $paramName, Type $originalType): bool
+    private function hasTypeChangedBeforeAssign(Assign $assign, string $paramName, Type $originalType, Scope $scope): bool
```

Comment:
Scope is pulled directly, so param `$scope` is not needed:

```suggestion
    private function hasTypeChangedBeforeAssign(Assign $assign, string $paramName, Type $originalType): bool
```